src/components/TimeRemaining.js

```diff
@@ -2,29 +2,69 @@ import React from 'react'
 import styled from 'styled-components'
 
 export default function TimeRemaining({ activities }) {
-  return remainingHours(activities) >= 1 ? (
-    <TimeLeftStyled>
-      You still have{' '}
-      {remainingHours(activities) % 1 !== 0
-        ? remainingHours(activities).toFixed(1)
-        : remainingHours(activities)}{' '}
-      {remainingHours(activities) === 1 ? 'hour' : 'hours'} to add to your day,
-      use {remainingHours(activities) === 1 ? 'it' : 'them'} wisely!
-    </TimeLeftStyled>
+  const timeRemaining = remainingHours(activities)
+
+  return timeRemaining >= 1 ? (
+    timeLeft()
+  ) : timeRemaining === 0 ? (
+    <NoTimeLeftStyled>
+      Day full! You don't have any hours remaining.
+    </NoTimeLeftStyled>
   ) : (
-    <TimeLeftStyled>You don't have any hours remaining</TimeLeftStyled>
+    <RemovehoursStyled>
+      You added too many hours to your day! Remove {hourOverflow(activities)}{' '}
+      hours or more before adding a new activity.
+    </RemovehoursStyled>
   )
 
+  function timeLeft() {
```

Comment:
You can pass in remainingHours as a parameter here, thereby making your function pure and indepentend of the TimeRemaining component. This is a great oportunity to create a new component `TimeLeft` that gets `remainingHours` as a prop.